[=== Module cuda/10.0 loaded ===]
[=== Module cuda/10.0/cudnn/7.6 loaded ===]
Traceback (most recent call last):
  File "run.py", line 76, in <module>
    app.run(main)
  File "/network/home/cheluver/miniconda3/envs/new/lib/python3.6/site-packages/absl/app.py", line 299, in run
    _run_main(main, args)
  File "/network/home/cheluver/miniconda3/envs/new/lib/python3.6/site-packages/absl/app.py", line 250, in _run_main
    sys.exit(main(argv))
  File "run.py", line 72, in main
    run_agent(FLAGS.env, agent, logs, aux_agent_configs, ignore_existent=False)
  File "/network/home/cheluver/jax_models/run_utils.py", line 14, in run_agent
    persistent_agent_config = configs.agent_config.config[agent["agent"]]
KeyError: 'mb_fw_iter'
